### PR TITLE
fix: Rename Infra documentation

### PR DESCRIPTION
**Pull request by CryptoKnightIOG**
https://input-output.atlassian.net/browse/ATL-7068

### CODE REVIEW

**Review by petevielhaber — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "prism-documentation-portal",
+  "name": "identus-documentation-portal",
```
Comment (mkbreuningIOHK) [inline package.json:2]: Is this file not generated automatically by the workflow (maybe this .github/workflows/build-docs.yml)? So I would not change this file.

**Comment by mkbreuningIOHK** [comment]
Let's have @milosbackonja review on this one.

File: .github/workflows/build-docs.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ defaults:
 
 env:
   REGISTRY: ghcr.io
-  IMAGE_NAME: prism-documentation-portal
+  IMAGE_NAME: identus-documentation-portal
```
Comment (yshyn-iohk) [inline .github/workflows/build-docs.yml:15]: @CryptoKnightIOG, could you involve @milosbackonja into image renaming.
I would recommend him to confirm that this renaming will work as expected.

File: .github/workflows/delete_old_packages.yml [hunk 3]
```diff
@@ -17,7 +17,7 @@ jobs:
       - uses: actions/delete-package-versions@v4
         with:
           owner: input-output-hk
-          package-name: prism-documentation-portal
+          package-name: identus-documentation-portal
           package-type: container
           min-versions-to-keep: 10
-          token: ${{ secrets.ATALA_GITHUB_TOKEN }}
+          token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (yshyn-iohk) [inline .github/workflows/delete_old_packages.yml:23]: This change should be reverted.
We can switch to the regular secrets.GITHUB_TOKEN after moving the repository to the Hyperledger.

**Review by milosbackonja — APPROVED** [review]

File: .github/workflows/delete_old_packages.yml (hunk 3 above)
Comment (CryptoKnightIOG) [reply .github/workflows/delete_old_packages.yml]: reverted

**Pull request merged**